### PR TITLE
feat: adding two CS related quiz questions

### PR DESCRIPTION
**Pull request by Aman0786-git**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by Aman0786-git** [comment]
This is my first contribution in an Open Source project . Please guide me if had made any mistake.Thankyou

File: src/data/full-quiz.ts [hunk 1]
```diff
@@ -7240,5 +7240,25 @@ const fullQuiz = [
...
     Link: "https://www.freecodecamp.org/news/scrum-the-hard-parts-2-sprint-harder/",
   },
+  {
+    Question:
+      "Choose the correct option?",
+    Answer: "Little O (o()) describes the upper bound excluding the exact bound",
+    Distractor1: "Big O (O()) describes the lower bound of the complexity.",
+    Distractor2: "Omega (Ω()) describes the upper bound of the complexity",
+    Distractor3: "Theta (Θ()) describes the lower bound excluding the exact bound",
+    Explanation: "Big O (O()) - Upper Bound ,Omega (Ω()) - Lower Bound , Theta (Θ()) - exact bound & Little O (o()) - describes the upper bound excluding the exact bound of the complexity ",
```
Comment (jdwilkin4) [inline src/data/full-quiz.ts:7250]: The explanation can be cleaned up a little bit. Let's just show the explanation for Little O. If they want to read more then they can click on the link for the article. 

```suggestion
    Explanation: "Little O (o()) describes the upper bound excluding the exact bound of the complexity ",
```

File: src/data/full-quiz.ts [hunk 2]
```diff
@@ -7240,5 +7240,25 @@ const fullQuiz = [
...
+    Link: "https://www.freecodecamp.org/news/big-o-notation-why-it-matters-and-why-it-doesnt-1674cfa8a23c/",
+  },
+  {
+    Question:
+      "Choose the correct option?",
+    Answer: "All of these",
+    Distractor1: "O(log(n)) is more complex than O(1), but less complex than polynomials",
+    Distractor2: "Complexity of polynomials increases as the exponent increases",
+    Distractor3: "O(1) has the least complexity",
+    Explanation: "O(1) often called as constant time is best for algorithms\nO(log(n)) is less complex than O(√n), because the square root function can be considered a polynomial, where the exponent is 0.5.\nHigher the power of n, more complext it becomes.For example, O(n⁵) is more complex than O(n⁴)",
```
Comment (jdwilkin4) [inline src/data/full-quiz.ts:7260]: The following explanation is a little long and would make the answer modal to crowded. Let's cut it down to just say all of the answers are correct. If the user wants to learn more, then they can click on the corresponding article. 

```suggestion
    Explanation: "All of the above options about Big O are correct. ",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Hi @Aman0786-git ! 
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Aman0786-git** [comment]
@jdwilkin4 Thanks for the review , I will make the changes .

**Comment by Aman0786-git** [comment]
@jdwilkin4 Thankyou will try to contribute regularly. :-)

**Pull request merged**